test/api-test/api/v2/collections/post.js

```diff
@@ -0,0 +1,58 @@
+// let expectedData is injected from <response-type>.json
+
+let responseData = pm.response.json();
+
+console.log("Check collection creation");
+
+pm.test("Check collection count", function () {
+  pm.expect(responseData.length).to.equal(expectedData.length);
+});
+
+for (let collectionIndex = 0; collectionIndex < expectedData.length; collectionIndex++) {
+  let expectedCol = expectedData[0];
+  let colNum = collectionIndex + 1;
+
+  let responseCol = responseData[0];
+  pm.test(`Collection ${colNum} - Check object type`, function () {
```

Comment:
The whitespace is weird here? Why aren't we indenting the anonymous function blocks?